src/lib/components/forms/Reprocess.svelte

```diff
@@ -86,8 +86,11 @@ This will mostly be used inside a modal but isn't dependent on one.
     const resp = await process(payload, csrf_token);
 
     if (resp.ok) {
-      await Promise.all(documents.map((d) => invalidate(`document:${d.id}`)));
       dispatch("close"); // closing destroys the component
+      await Promise.all([
```

Comment:
By moving the `await` beneath the `dispatch("close")`, won't the component be destroyed before the invalidations can finish?

Reply:
Very possibly. I was struggling with invalidation and this might be at the heart of it. Will test both ways before merging.